README.md

```diff
@@ -14,4 +14,4 @@ The `App`contains a `MoviesList` and a search field. Implement filtering using a
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_movies-list-filter/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://katerynashylina.github.io/react_movies-list-filter/) and add it to the PR description.
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.